expression/builtin_string_vec.go

```diff
@@ -652,11 +651,21 @@ func (b *builtinLeftBinarySig) vecEvalString(input *chunk.Chunk, result *chunk.C
 }
 
 func (b *builtinReverseBinarySig) vectorized() bool {
-	return false
+	return true
 }
 
 func (b *builtinReverseBinarySig) vecEvalString(input *chunk.Chunk, result *chunk.Column) error {
-	return errors.Errorf("not implemented")
+	if err := b.args[0].VecEvalString(b.ctx, input, result); err != nil {
+		return err
+	}
+	for i := 0; i < input.NumRows(); i++ {
+		if result.IsNull(i) {
+			continue
+		}
+		reversed := reverseBytes(result.GetBytes(i))
+		result.SetRaw(i, []byte(string(reversed)))
```

Comment:
Why transform `[]byte` to `string`, and then transform `string` to `[]byte`? I think it is useless.

Reply:
One more thing, we really need call the `SetRaw` function?

Reply:
Why the non-vector fuc used this method?
Could you tell me what fuc should I use please?

Reply:
It may have better performance by using `SetRaw`.
@icditwang , Could you benchmark it again and update your benchmark result into PR description?

Reply:
> Why transform `[]byte` to `string`, and then transform `string` to `[]byte`? I think it is useless.

And please resovle this issue.

Reply:
@SunRunAway I updated it,it's fuster.